docs/cmd/tkn_clustertask_list.md

```diff
@@ -19,7 +19,7 @@ Lists clustertasks in a namespace
 ```
       --allow-missing-template-keys   If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats. (default true)
   -h, --help                          help for list
-  -o, --output string                 Output format. One of: json|yaml|name|template|go-template|go-template-file|templatefile|jsonpath|jsonpath-file.
+  -o, --output string                 Output format. One of: json|yaml|name|go-template|go-template-file|templatefile|template|jsonpath|jsonpath-file.
```

Comment:
This seems like unintentional change and happening due to undetermined sorting, we faced this in knative/client too
https://github.com/knative/client/issues/135

@rhuss fixed this by updating the cli-runtime dependency
https://github.com/knative/client/pull/137/files#diff-31be3fbc3cb3d1a36172f2111034b4ceR247

If there isnt any conflict, this can be updated 
https://github.com/tektoncd/cli/blob/8dfeaa4f4a50c69a89011c0badfaeadcd81e8f42/vendor/modules.txt#L443

Reply:
@navidshaikh Does the second commit look good?

Reply:
@piyush-garg : TBH, its good to have a separate PR fixing this, have that merged and then your resource describe command PR shouldn't have unnecessary changes.

Reply:
sure, makes sense, will raise PR

Reply:
raised #175

Reply:
Thanks, @navidshaikh for pointing out and guiding. Issue resolved in #175 and this PR now looks better. Thanks a lot